addons/mail/static/src/messaging/entity/message/message_tests.js

```diff
@@ -0,0 +1,114 @@
+odoo.define('mail.messaging.entity.MessageTests', function (require) {
+'use strict';
+
+const {
+    afterEach: utilsAfterEach,
+    beforeEach: utilsBeforeEach,
+    pause,
+    start: utilsStart,
+} = require('mail.messaging.testUtils');
+
+QUnit.module('mail', {}, function () {
+QUnit.module('messaging', {}, function () {
+QUnit.module('entity', {}, function () {
+QUnit.module('Message', {
+    beforeEach() {
+        utilsBeforeEach(this);
+        this.start = async params => {
+            if (this.widget) {
+                this.widget.destroy();
+            }
+            let { env, widget } = await utilsStart(Object.assign({}, params, {
+                data: this.data,
+            }));
+            this.env = env;
+            this.widget = widget;
+        };
+    },
+    afterEach() {
+        utilsAfterEach(this);
+        this.env = undefined;
+        if (this.widget) {
+            this.widget.destroy();
+            this.widget = undefined;
+        }
```

Comment:
This is a general comment, not only on this file, but I think usually destroy/after methods should apply their operations in the reverse of the start/before.

So here I would call `utilsAfterEach` at the end.